config/software/chef.rb

```diff
@@ -55,14 +55,12 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  # compiled ruby on windows 2k8R2 x86 is having issues compiling
-  # native extensions for pry-byebug so excluding for now
-  excluded_groups = %w{server docgen maintenance pry travis integration ci chefstyle debug}
```

Comment:
This is probably fine now that we're not building chef 14 again

Reply:
I assume pry won't be an issue on modern windows

Reply:
we'll have to find out.  the right thing to do is ship it here and appbundle it to avoid conflicts with the pinned stuff in inspec (reminder of why appubndler exists and pins the entire world).